app/logic/util/convertHTML.ts

```diff
@@ -133,6 +150,12 @@ DOMPurify.addHook('afterSanitizeAttributes', node => {
           node.setAttribute(attribute, "");
           node.setAttribute("title", ex.message ?? ex + "");
         }
+      } else if ((node.tagName.toLocaleLowerCase() == "img" && attribute == "src") ||
+          (node.tagName.toLocaleLowerCase() == "link" && node.getAttribute("rel") == "stylesheet" && attribute == "href")
+          && includeExternal) {
+        let url = node.getAttribute(attribute);
+        let newURL = externalSrc(url);
```

Comment:
Given that this no longer allows `data:` and `cid:` URLs, it now breaks inline images.